frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -164,6 +164,11 @@ export const BillingProductAddon = ({ addon }: { addon: BillingProductV2AddonTyp
                                 type="primary"
                                 icon={<IconPlus />}
                                 size="small"
+                                disabledReason={
```

Comment:
There should be a [billingloading reducer](https://keajs.org/docs/plugins/loaders/) we can use to determine if the deactivateproduct loader that's called upon unsubscribe in the `frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx` has completed. Would that help?

Reply:
Oh thank you! I will look deeper into this.